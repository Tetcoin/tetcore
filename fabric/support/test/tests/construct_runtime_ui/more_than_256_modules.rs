use fabric_support::construct_runtime;

construct_runtime! {
	pub enum Runtime where
		UncheckedExtrinsic = UncheckedExtrinsic,
		Block = Block,
		NodeBlock = Block,
	{
		System: system::{} = 255,
		Noble256: noble256::{},
	}
}

fn main() {}
