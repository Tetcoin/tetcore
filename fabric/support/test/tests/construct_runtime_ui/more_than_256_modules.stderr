error: Module index doesn't fit into u8, index is 256
  --> $DIR/more_than_256_modules.rs:10:3
   |
10 |         Noble256: noble256::{},
   |         ^^^^^^^^^
